closure/goog/events/eventid.js

```diff
@@ -18,15 +18,15 @@ goog.provide('goog.events.EventId');
 
 /**
  * A templated class that is used when registering for events. Typical usage:
- * <code>
- *   /** @type {goog.events.EventId<MyEventObj>}
+ * <pre>
```

Comment:
Do you still need the entity encoding if you're using 4 indents? I assumed
the encoding was only to fix the generated API.

On Mon, Nov 16, 2015 at 3:20 PM, Erik Neumann <notifications@github.com>
wrote:

> In closure/goog/events/eventid.js
> <https://github.com/google/closure-library/pull/603#discussion_r45000355>:
>
> > @@ -18,15 +18,15 @@ goog.provide('goog.events.EventId');
> >
> >  /**
> >   * A templated class that is used when registering for events. Typical usage:
> > - * <code>
> > - *   /** @type {goog.events.EventId<MyEventObj>}
> > + * <pre>
>
> Unfortunately that seems to not work: then the &gt; does not show up as <
> in the docs. See #603 (comment)
> <https://github.com/google/closure-library/pull/603#issuecomment-156789878>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/closure-library/pull/603/files#r45000355>.
>